wp/sections/compare.tex

```diff
@@ -0,0 +1,34 @@
+\subsection{About Ktlint}
+\par Ktlint is a popular an anti-bikeshedding Kotlin linter with built-in formatter. It tries to capture (reflect) official code style from kotlinlang.org and Android Kotlin Style Guide and then automatically apply these rules to your codebase. It claims to be simple and easy to use. Ktlint has been developing since 2016 and from then on it has 3.8k stars, 299 forks and 390 closed PRs (at least on the moment of writing this whitepaper). There have been written over 15k lines of code. Ktlint has it's own ruleset, which divides on standard and experimental rules. Ktlint can be used as a plugin via Maven or Gradle.
+
+\subsection{About Detekt}
+\par Detekt is a static code analysis tool. It operates on an abstract syntax tree provided by Kotlin compiler. Detekt supports such features as code smell analysis, highly configurable rule sets, IntelliJ integration and third-party integrations for Maven, Bazel and Github actions and many more. It has been developing since 2016 and today it has 3.2k stars, 411 forks and 1850 closed PRs. It has around 45k lines of code.
```

Comment:
as I was told in ktlint - detekt is using it for some of rules. 
Could you please check it and write it here?

Reply:
Yes, detekt has a whole group of rules, `detekt-formatting-rules`, that are wrappers around ktlint rules